test/commands/git/checkout.test.ts

```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@oclif/test'
+
+describe.skip('git:checkout', () => {
```

Comment:
Just my opinion. Lack of unit tests should be an auto fail of a PR.

Reply:
Agreed. Also sounds like a good contribution to this PR to validate the understanding and familiarize with the tools/frameworks.